elasticassignment/ageofempires/management/commands/command_classes.py

```diff
@@ -0,0 +1,206 @@
+                }
+            },
+            'mappings': {
+                'properties': {
+                    "army" : {"type" : "text"},
+                    "civilization_bonus" : {"type" : "text"},
+                    "id" : {"type" : "integer"},
+                    "name" : {"type" : "text"},
+                    "team_bonus" : {"type" : "text"},
+                    "unique_tech" : {"type" : "text"},
+                    "unique_unit" : {"type" : "text"}
+                }
+            }
+        }
+        print("creating 'civilization' index...")
+        Civilizations.es.indices.create(index = 'civilization', body = request_body)
+
+    def populate_index(url:str):
+        return_list = AOElasticsearch.getAPIdata(url)
+        civilizations = return_list['civilizations']
+        civilization_list = []
+        for civlization in civilizations:
+            civ = {
+                "_index": "civilization",
+                "_source": civlization
+            }
+            civilization_list.append(civ)
+        bulk(Civilizations.es, civilization_list, 20)
+
+
+class Structures(AOElasticsearch):
+    es = AOElasticsearch.make_connection()
+    def make_index():
+        request_body = {
+            "settings" : {
+                "index":{
+                    "number_of_shards": settings.NUMBER_OF_SHARDS,
+                    "number_of_replicas": settings.NUMBER_OF_REPLICAS
+                }
+            },
+            'mappings': {
+                'properties': {
+                    "age" : {"type" : "text"},
+                    "armor" : {"type" : "text"},
+                    "attack" : {"type" : "integer"},
+                    "build_time" : {"type" : "integer"},
+                    "description" : {"type" : "text"},
+                    "expansion" : {"type" : "text"},
+                    "hit_points" : {"type" : "integer"},
+                    "id" : {"type" : "integer"},
+                    "line_of_sight" : {"type" : "integer"},
+                    "name" : {"type" : "text"},
+                    "reload_time" : {"type" : "double"},
+                    "special" : {"type" : "text"},
+                    "structure_range" : {"type" : "text"}
+                    }    
+                }
+            }
+        print("creating 'structures' index...")
+        Structures.es.indices.create(index = 'structures', body = request_body)
+
+    def populate_index(url:str):
+        return_list = AOElasticsearch.getAPIdata(url)
+        structures = return_list['structures']
+        structures_list = []
+        for structure in structures:
+            if 'range' in structure:
+                structure['structure_range'] = structure['range']
+                del structure['range']
+            struc = {
+                "_index": "structures",
+                "_source": structure
+            }
+            structures_list.append(struc)
+        bulk(Structures.es, structures_list, 20)
```

Comment:
as explained, please don't use the batch size param in `bulk` function in this particular exercise.
The batch size is to be controlled by your python program.